internal/services/network/public_ip_resource_test.go

```diff
@@ -740,12 +740,13 @@ resource "azurerm_resource_group" "test" {
 }
 
 resource "azurerm_public_ip" "test" {
-  name                = "acctestpublicip-%d"
-  location            = azurerm_resource_group.test.location
-  resource_group_name = azurerm_resource_group.test.name
-  allocation_method   = "Static"
-  sku                 = "Basic"
-  domain_name_label   = "acctest-%d"
+  name                    = "acctestpublicip-%d"
+  location                = azurerm_resource_group.test.location
+  resource_group_name     = azurerm_resource_group.test.name
+  allocation_method       = "Static"
+  sku                     = "Basic"
+  domain_name_label       = "acctest-%d"
+  idle_timeout_in_minutes = 30
```

Comment:
Can we add one more step in the `TestAccPublicIpStatic_idleTimeout` test case to only update the `idle_timeout_in_minutes`?